src/openms/source/TRANSFORMATIONS/FEATUREFINDER/FeatureFinderAlgorithmPicked.cpp

```diff
@@ -358,7 +359,8 @@ namespace OpenMS
             {
               Size spec_index = next_spectrum.findNearest(pos);
               double position_score = positionScore_(pos, next_spectrum[spec_index].getMZ(), trace_tolerance_);
-              if (position_score > 0 && next_spectrum[spec_index].getIntensity() > intensity) is_max_peak = false;
+              if (position_score > 0 && next_spectrum[spec_index].getIntensity() > intensity)
+                is_max_peak = false;
```

Comment:
this here should be the last remaining case. There are a few more unnecessary style and whitespace changes, but I think they are fine.

Reply:
(another commit should also restart the tests, some of which had problems during the weekend)